lib/sensors/stripe_sensor.cpp

```diff
@@ -0,0 +1,26 @@
+#include "stripe_sensor.hpp"
+
+namespace hyped::sensors {
+
+Keyence::Keyence(core::ILogger &log, const std::uint8_t newPin) : pin(newPin), logger_(log)
+{
+  stripe_count_ = 0;
+  io::HardwareGpio hardware(logger_);
+  keyence = hardware.getReader(pin);
+};
+
+Keyence::~Keyence()
+{
+}
+
+int Keyence::getStripeCount()

```

Comment:
The return type of this function is `int` but `stripe_count_` is `uint16_t` so you'll have to change the return type here.